skyportal/model_util.py

```diff
@@ -10,6 +10,7 @@
     'Manage users',
     'Manage sources',
     'Manage groups',
+    'Manage shifts',
```

Comment:
Also add "Manage shifts" to the "Group admin" role below (line 24). I think that's the default behavior you want...

Reply:
added